cms/djangoapps/contentstore/tests/test_video_utils.py

```diff
@@ -389,7 +389,7 @@ def test_video_backend(self):
     @override_settings(VIDEO_IMAGE_SETTINGS={
         'STORAGE_CLASS': 'storages.backends.s3boto3.S3Boto3Storage',
         'STORAGE_KWARGS':
-            {'bucket_name': 'test', 'default_acl': None, 'base_url': '/', 'location': 'abc/def'}}
+            {'bucket_name': 'test', 'default_acl': None, 'location': 'abc/def'}}
```

Comment:
Constructor now verifies its passing params and base_url is unknown to this package.